Update index.md Added section on Limitations (fixed typo) (#4802)
segmentio/segment-docs: src/connections/destinations/catalog/actions-qualtrics/index.md

<!-- The section below explains how to enable and configure the destination. Include any configuration steps not captured below. For example, obtaining an API key from your platform and any configuration steps required to connect to the destination. -->

### Limitations

> At this time, Qualtrics as a destination is only available to customers on a Business Tier plan with Segment.
> 
> To link your Qualtrics destination in Segment to your Qualtrics workspace, [Qualtrics]([url](https://www.qualtrics.com/support/integrations/twilio-segment/twilio-segment-task/)) requires a Segment Token ([Public API Token]([url](https://app.segment.com/goto-my-workspace/settings/access-management/tokens))) that can only be generated by Business Tier customers, who have access to the [Public API]([url](https://segment.com/docs/api/public-api/)).

Reviewer: ```suggestion
The Qualtrics destination is only available to customers on a Business Tier plan with Segment.

To link your Qualtrics destination in Segment to your Qualtrics workspace, [Qualtrics](https://www.qualtrics.com/support/integrations/twilio-segment/twilio-segment-task/){:target="_blank"} requires a Segment Token [Public API Token](https://app.segment.com/goto-my-workspace/settings/access-management/tokens){:target="_blank"} that can only be generated by Business Tier customers, who have access to the [Public API](https://segment.com/docs/api/public-api/){:target="_blank"}.
```

> To link your Qualtrics destination in Segment to your Qualtrics workspace, [Qualtrics](https://www.qualtrics.com/support/integrations/twilio-segment/twilio-segment-task/){:target="_blank"} requires a Segment Token [Public API Token](https://app.segment.com/goto-my-workspace/settings/access-management/tokens){:target="_blank"} that can only be generated by Business Tier customers, who have access to the [Public API](https://segment.com/docs/api/public-api/){:target="_blank"}.

Reviewer: ```suggestion
To link your Qualtrics destination in Segment to your Qualtrics workspace, [Qualtrics](https://www.qualtrics.com/support/integrations/twilio-segment/twilio-segment-task/){:target="_blank"} requires a [Segment Token](https://app.segment.com/goto-my-workspace/settings/access-management/tokens){:target="_blank"} that can only be generated by Business Tier customers, who have access to the [Public API](https://segment.com/docs/api/public-api/){:target="_blank"}.
```
